### DISCUSSION TITLE
Github Followers

### DISCUSSION THREAD

**Discussion by itsdotnet** [community/community#68506] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Guys what do you think how to get more followers on github.

**Comment by Jonak-Adipta-Kalita** [comment]
Make good projects, which can be helpful to people, so that they like your product, and follow you!

**Comment by mantasu** [comment]
I think people will follow you for several reasons:
1. **Quality**: whatever you contribute, be it projects, pull requests, discussions, reviews, etc., it should be of exceptional quality. If you find yourself excelling in one of the areas (e.g., Web Development Cheatsheets) try contributing to it and people will follow you for your expertise. But remember that the quality of the content also matters: it must be well structured, documented, and easy to read/follow.
2. **Relevance**: try aiming to contribute to projects that are most relevant to the community - the more people there are, the more chances there will be someone interested in your work
3. **Activity**: the more active you are, the more passionate and determined you will look. Also, if you are active in the communities, you will get more chances of being noticed
4. **Consistency**: be consistent with your work and contributions. People who follow you may expect to hear from you on a regular basis. For instance, if you upload a summary of some interesting paper monthly, stick to it, don't do some months and skip others.
5. **Interest**: show your interest in others as well, for example, if you follow other people, there are chances that they will follow you back, especially if you have projects or goals in common
